src/Microsoft.ML.StandardLearners/Standard/SdcaBinary.cs

```diff
@@ -1574,8 +1509,131 @@ private protected override void CheckLabel(RoleMappedData examples, out int weig
             weightSetCount = 1;
         }
 
-        protected override BinaryPredictionTransformer<TScalarPredictor> MakeTransformer(TScalarPredictor model, Schema trainSchema)
-            => new BinaryPredictionTransformer<TScalarPredictor>(Host, model, trainSchema, FeatureColumn.Name);
+        protected override BinaryPredictionTransformer<TModelParameters> MakeTransformer(TModelParameters model, Schema trainSchema)
+            => new BinaryPredictionTransformer<TModelParameters>(Host, model, trainSchema, FeatureColumn.Name);
+    }
+
+    public sealed class SdcaCalibratedBinaryTrainer :
+        SdcaBinaryTrainerBase<CalibratedModelParametersBase<LinearBinaryModelParameters, PlattCalibrator>>
+    {
+        internal const string LoadNameValue = "SDCALR";
+        internal const string UserNameValue = "Fast Linear (SA-SDCA) Logistic Regression";
+
+        /// <summary>
+        /// Configuration to training logistic regression using SDCA.
+        /// </summary>
+        public sealed class Options : BinaryArgumentBase
+        {
+        }
+
+        internal SdcaCalibratedBinaryTrainer(IHostEnvironment env,
+            string labelColumn = DefaultColumnNames.Label,
```

Comment:
>labelColumn [](start = 19, length = 11)

someone once suggested those should be called labelColumnName :)

Reply:
I shouldn't copy-and-paste..

---
In reply to: [256257727](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256257727) [](ancestors = 256257727)